iroh/src/commands/node.rs

```diff
@@ -28,7 +28,7 @@ use super::{BlobAddOptions, MAX_RPC_CONNECTIONS, MAX_RPC_STREAMS};
 #[derive(Debug)]
 pub struct StartOptions {
     pub addr: SocketAddr,
-    pub rpc_port: RpcPort,
+    pub rpc_port: u16,
```

Comment:
So we can't start with rpc port disabled anymore?

Reply:
For now I have removed the ability, yes. Lets work on getting the portless version for rpc supported soon to improve on this.